README.Mac.md

```diff
@@ -1,294 +1,281 @@
-Since the install script was created for LINUX OS, its apache2 setup steps won’t work. Section 3 describes how to set up the apache2 server for Mac.
+## 1.6 Install PHP and pk-config
 
-## 2.3 Manually running install script steps
+Paste these commands into your terminal while hitting enter:
+```
+brew install php72
+brew link php72
+brew services start php72
+brew install pkg-config
+```
 
-Since you may not be able to fully execute the install.sh script at step 2.2, the following examples may help in manually executing the sequence of commands found in the install script.
+### 1.6.1 Modify Apache Configuration (again) but for PHP
 
-Set up these variables in your bash shell :
-``` 
-mysqldb=LORIS
-mysqlhost=localhost
-mysqluser=lorisuser  (suggested)
-mysqlpass=password 
-mysqlrootuser=root 
+Modify the apache configuration file to enable php.
+```
+sudo open -e /usr/local/etc/httpd/httpd.conf
 ```
-Keep this terminal window open to keep the variables in memory.
 
-Then execute the following commands in your bash shell to create the Loris database and the lorisuser MySQL user. 
-The prompt should ask for the root password. (The one entered in section 1.5.4) 
+**a.** Add the below line to LoadModules (Change php@<version> to the correct version):
 ```
-echo "CREATE DATABASE $mysqldb" | mysql -h$mysqlhost --user=$mysqlrootuser -p -A
-echo "GRANT UPDATE,INSERT,SELECT,DELETE ON $mysqldb.* TO '$mysqluser'@'localhost' IDENTIFIED BY '$mysqlpass' WITH GRANT OPTION" | mysql $mysqldb -h$mysqlhost --user=$mysqlrootuser -p -A
+LoadModule php7_module /usr/local/opt/php@<version>/lib/httpd/modules/libphp7.so
 ```
 
-Source the database schema with the following statement in your bash shell (the terminal window used at section 1.5.4) :
+**b.** Modify <IfModule dir_module> to this:
 ```
-mysql $mysqldb -h$mysqlhost --user=$mysqlrootuser -p -A 2>&1 < ~/Sites/$projectname/SQL/0000-00-00-schema.sql
+<IfModule dir_module>
+    DirectoryIndex index.php index.html
+</IfModule>
 ```
 
-# 3. Launching the Local Apache2 Server
-This section goes into details regarding setting up the local Apache2 server. There are Max OS X 10.10 (Yosemite) specific instructions in section 3.1. If you are not running Yosemite step 3.1 may be be skipped.
-
-## 3.1 Mac OS X Yosemite specific configuration (for Yosemite users only)
-
-In ```/etc/apache2/httpd.conf``` locate the following lines and uncomment them (by removing the ``` # ```) if they are not commented out:
-
-```#LoadModule userdir_module libexec/apache2/mod_userdir.so ```
-and
-``` #Include /private/etc/apache2/extra/httpd-userdir.conf ```
+**c.** Add the below lines under </IfModule>:
+```
+<FilesMatch \.php$>
+    SetHandler application/x-httpd-php
+</FilesMatch>
+```
 
-At the end of your file also include the line
-``` Include /etc/apache2/other/*.conf ```
+#### Restart apache and it will reload the updated settings
```

Comment:
modify to "Restart apache **with** the updated settings"

Reply:
```suggestion
#### Restart Apache will the updated settings
```